### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by pouya-p** [projectdiscovery/shuffledns#202] [Q&A]
i installed massdns and go , than installed shuffledns  but isnt working.

--/go/pkg/mod/github.com/projectdiscovery# massdns -h
Usage: massdns [options] [domainlist]--

--go/pkg/mod/github.com/projectdiscovery# go version
go version go1.20.2 linux/amd64--

--/go/pkg/mod/github.com/projectdiscovery# shuffledns -h
shuffledns: command not found--

please help

**Comment by MetzinAround** [comment]
@pouya-p it might have to do with your PATH. Make sure the directory where shuffledns is located is in your PATH. 


Also, give dnsx a try as well. http://nux.gg/dnsx
